Makefile

```
@@ -237,6 +237,12 @@ dev:  ## Run the development server in a Docker container.
 	@echo "███ Starting development server..."
 	@OFFSET_PORTS='false' DOCKER_BUILD_VERBOSE='true' $(DEVSHELL) $(SDBIN)/run
 	@echo
+	
+.PHONY: dev-detatched
+dev-detatched:  ## Run the development server in a Docker container without attatching tty.
+	@echo "███ Starting development server..."
+	@OFFSET_PORTS='false' DETATCHED='true' DOCKER_BUILD_VERBOSE='true' $(DEVSHELL) $(SDBIN)/run
```

Typos:

```suggestion
dev-detached:  ## Run the development server in a Docker container without attaching tty.
	@echo "███ Starting development server..."
	@OFFSET_PORTS='false' DETACHED='true' DOCKER_BUILD_VERBOSE='true' $(DEVSHELL) $(SDBIN)/run
```

But it doesn't appear that `DETATCHED` is referenced anywhere?

```sh-session
(.venv) user@sd-dev:~/securedrop$ grep -R "DETATCHED" *
Makefile:	@OFFSET_PORTS='false' DETATCHED='true' DOCKER_BUILD_VERBOSE='true' $(DEVSHELL) $(SDBIN)/run
```